### DISCUSSION TITLE
notes/tools/draw.io/

### DISCUSSION THREAD

**Discussion by giscus** [chrisxfire/chrisxfire.github.io#250] [Comments]
# notes/tools/draw.io/

chrisxfire blog

https://chriscodes.net/notes/tools/draw.io/

**Comment by chrisxfire** [comment]
Canvas, toggle grid, Ctrl + Shift + G

**Comment by chrisxfire** [comment]
Shapes, Group, Ctrl +G
Shapes, Ungroup, Ctrl + Shift + U
